src/clientmetricreport/ClientMetricReport.ts

```diff
@@ -329,6 +373,22 @@ export default class ClientMetricReport {
     }
   }
 
+  /**
+   *  Dimensions derived from metric
+   */
+  readonly streamMetricDimensionMap: {
```

Comment:
Nit: Don't think we need 'Metric' in any of this or related naming?

Reply:
Updated naming as suggested.